content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst

```diff
@@ -28,30 +28,31 @@ three step deliveries. First, make sure the *Multi-Step Routes* option is enable
    :alt: Activate multi-step routes and storage locations in inventory settings.
 
 Next, the warehouse needs to be configured for three step deliveries. To do this, go to
-:menuselection:`Inventory --> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the
-:guilabel:`Warehouse`. Then, select :guilabel:`Pack goods, send goods in output and then deliver
-(3 steps)` for :guilabel:`Outgoing Shipments`. Then :guilabel:`Save` the changes.
+:menuselection:`Inventory app --> Configuration --> Warehouses`, and click on the warehouse to edit.
```

Comment:
I feel that **warehouse** may need a guilabel since it is part of the active instruction
```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouses`, and click on the :guilabel:`warehouse` to edit.
```